release-doc-in-process/all_vf.TTL

```diff
@@ -82,26 +88,28 @@ vf:EconomicEvent  a            owl:Class ;
         rdfs:label       "vf:EconomicEvent" ;
         rdfs:comment   "An inflow or outflow of an economic resource in relation to a process and/or exchange. This could reflect a change in the quantity of a vf:EconomicResource. It is also defined by its behavior in relation to the vf:EconomicResource and a vf:Process (consume, use, produce, etc.)" .
 
-vf:Agent  a                  owl:Class ;
-        rdfs:label           "vf:Agent" ;
-        owl:equivalentClass  foaf:Agent ;
-        rdfs:comment   "A person or group with economic agency." .
+foaf:Agent  a                  owl:Class ;
```

Comment:
what's the motivation for using `foaf` / `org` directly instead of subclassing as `vf`? i'm happy if there's a link to a discussion.